test/gurka/test_recost.cc

```diff
@@ -7,6 +7,57 @@ using namespace valhalla;
 
 const std::unordered_map<std::string, std::string> build_config{{"mjolnir.shortcuts", "false"}};
 
+TEST(recosting, forward_vs_reverse) {
```

Comment:
I tried to write a gurka test for bidir-a* for a few hours but it's too tough, sorry

Reply:
correct. it's red on master and green with after the fix